backend/models/user.py

```diff
@@ -57,4 +56,14 @@ def get_user(cls, uuid: str):
         try:
             return User.from_dict(u)
         except Exception as e:
-            raise UserCreationError(u) from e
\ No newline at end of file
+            raise UserCreationError(u) from e
+
+    def delete_user(self):
```

Comment:
Great! I understand that it is just for testing purposes but keep in mind that user uuid is also stored in plan model and maybe would be great to delete there too?

Reply:
All the data is removed when the tests finished, so I am not going to do that implementation for "testing". I am sorry

Reply:
sure, no problem 😄